examples/deployment/algob.config.js

```diff
@@ -52,6 +53,28 @@ let accounts = mkAccounts([
   }
 ]);
 
+// KMD credentials
+let KMDConfig = {
```

Comment:
You call this KMDConfig but add it later as a network.
Isn't it that the network uses one port and KMD has it's own?

Reply:
yes, i was trying to include it in env, so that KMD token is accessible from env.

Reply:
One more thing for same line: local variables should be lower case (unless we define exported constants - which then should be upper case)